mkpkg

```diff
@@ -118,10 +118,11 @@ class Functions:
         debugf("Output: {}", output)
         return regex.findall(output)
 
-    def autodeps(self, fname):
+    def autodeps(self, *fnames):
```

Comment:
It's `(T...)` in D lingo. Variable arguments, where `fname` ends up with a list of arguments.
```py
autodepts(x, y, z) -> fnames == [x, y, z]
```
On call site, it converts an *iterable* to a list of arguments:
```py
l = [x, y, z]
autodepts(l) -> fnames == [l] -> fnames == [[x,y,z]]
autodepts(*l) -> fnames == [x, y, z]
```